pkg/util/wait/wait.go

```diff
@@ -82,22 +81,18 @@ func JitterUntil(f func(), period time.Duration, jitterFactor float64, sliding b
 		}
 
 		func() {
-			defer runtime.HandleCrash()
```

Comment:
This change is from: 

https://godoc.org/k8s.io/kubernetes/pkg/util/runtime 

```
TODO: remove this function. We are switching to a world where it's safe for apiserver to panic, since it will be restarted 
by kubelet. At the beginning of the Kubernetes project, nothing was going to restart apiserver and so catching panics 
was important. But it's actually much simpler for montoring software if we just exit when an unexpected panic happens.
```